Stratis.Bitcoin/RPC/Controllers/FullNodeController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using NBitcoin;
+using Stratis.Bitcoin.RPC.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Stratis.Bitcoin.RPC.Controllers
+{
+	//TODO: Need to be extensible, should be FullNodeController
+	public partial class ConsensusController : Controller
+	{
+		[ActionName("stop")]
+		public Task Stop()
+		{
+			_FullNode.Dispose();
+			return Task.CompletedTask;
+		}
+
+		[ActionName("getrawtransaction")]
+		public async Task<TransactionModel> GetRawTransaction(string txid, int verbose = 0)
+		{
+			uint256 trxid;
+			if (!NBitcoin.uint256.TryParse(txid, out trxid))
+				throw new ArgumentException(nameof(txid));
+
+			ChainedBlock block = null;
+			Transaction trx = (await _FullNode.MempoolManager.InfoAsync(trxid))?.Trx;
```

Comment:
Yes, I relied on the store check.
I can put Null checks on these. Because this controller requires FullNode, I assumed the store and mempool would have to be available. I'll put these checks in for now. In the future I'd like to see the controller for these RPC actions obtain available services themselves, and filter behavior on what's available.